docs/superpowers/specs/assets/2026-06-03-cli-device-login-flow-diagram.html

```
@@ -0,0 +1,121 @@
+    </pre>
+  </section>
+
+  <!-- Simple block view -->
+  <section class="card mt-6 p-6">
+    <h2 class="mb-4 text-sm font-bold tracking-[0.18em] text-slate-500 uppercase">At a glance — the two credentials</h2>
+    <pre class="mermaid">
+flowchart LR
+    Dev([Developer]) -->|runs capgo login| CLI[CLI · terminal]
+    CLI <-->|device_code secret<br/>start + poll| BE[(Capgo Backend)]
+    CLI -->|opens browser| Web[Browser page]
+    Web <-->|JWT session<br/>session + authorize| BE
```

HTMLHint (1.9.2)

[error] Tag must be paired, missing: [ </--></--> ], start tag match failed [ <--> ] on line 88.

(tag-pair)